VAMobile/src/components/Templates/FeatureLandingAndChildTemplate.tsx

```diff
@@ -143,6 +143,8 @@ export const ChildTemplate: FC<ChildTemplateProps> = ({
           transitionHeader(event.nativeEvent.contentOffset.y)
         }}
         {...scrollViewProps}>
+        {/* not sure on this one */}
```

Comment:
This is just the main body content block right? If so, I think we're fine to leave it to the a11y label